.jenkins/static.groovy

```diff
@@ -18,11 +18,10 @@ def runCI =
 
     if (env.BRANCH_NAME ==~ /PR-\d+/ && pullRequest.labels.contains("noSolver"))
     {
-        prj.libraryDependencies = ['rocBLAS', 'hipBLAS-common']
-    }
+        prj.libraryDependencies = ['hipBLAS-common', 'hipBLASLt', 'rocBLAS']    }
     else
     {
-        prj.libraryDependencies = ['rocBLAS', 'rocSPARSE', 'rocSOLVER', 'rocPRIM', 'hipBLAS-common']
+        prj.libraryDependencies = ['hipBLAS-common', 'hipBLASLt', 'rocBLAS', 'rocSPARSE', 'rocSOLVER',]
```

Comment:
rocPRIM should stay if solver still depends on it ?    And maybe put in all 3 files...

Reply:
you are correct. I just committed the new changes.